dockers/docker-orchagent/supervisord.conf

```diff
@@ -62,9 +74,8 @@ dependent_startup_wait_for=orchagent:running
 command=/usr/bin/swssconfig.sh
 priority=6
 autostart=false
-autorestart=unexpected
+autorestart=false
```

Comment:
i am not sure why you want to change this, can you explain the reason?

Reply:
Based on desire to not restart. If we fail, I don't want to flap. Ok for the switch to not come up so we can see the problem, vs constantly coming up, down, up, down.

Reply:
how is related to gearbox setup?

Reply:
@slgan621 move it out.